src/main/java/core/jdbc/JdbcTemplate.java

```diff
@@ -1,52 +1,58 @@
 package core.jdbc;
 
-import java.sql.Connection;
-import java.sql.PreparedStatement;
-import java.sql.ResultSet;
-import java.sql.SQLException;
+import org.springframework.dao.IncorrectResultSizeDataAccessException;
+import org.springframework.jdbc.core.PreparedStatementSetter;
 
-public class JdbcTemplate {
+import java.util.List;
 
-    public static void insert(String query, Object... values) throws SQLException {
+public class JdbcTemplate extends AbstractJdbcTemplate{
+
+    public void insert(String query, Object... values) throws DataAccessException {
         modify(query, values);
     }
 
-    public static void update(String query, Object... values) throws SQLException {
+    public void insert(String query, PreparedStatementSetter setter) throws DataAccessException {
+        modify(query, setter);
```

Comment:
`modify()` 메서드로 랩핑을 한 이유가 혹시 있으신가요?
지금은 별 다른 로직이 없는데, `connectionAndModify()` 를 바로 호출하면 안되나요?
```suggestion
        connectionAndModify(query, setter);
```